platform-tests/src/test/java/org/junit/AutomaticModuleNameTests.java

```diff
@@ -59,6 +59,7 @@ private static String version(String module) {
 		throw new AssertionError("module name is unknown: " + module);
 	}
 
+	@SuppressWarnings("FilesLinesLeak") // stream closed indirectly by JUnit 5
```

Comment:
At least, I _think_ JUnit 5 automatically handles closing of Streams. I could be wrong.

If that's the case, then I think a new issue would need to be raised, as I'd be out of my league.